<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonManager</class>
 <widget class="QMainWindow" name="ButtonManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duchar Favorites</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/fav</normaloff>:/icons/fav</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QListWidget" name="buttonList">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/close</normaloff>:/icons/close</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/minus</normaloff>:/icons/minus</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
